codelab-master/apps/playground/src/app/quiz/containers/introduction/introduction.component.html

```diff
@@ -0,0 +1,40 @@
+<ng-container *ngFor="let quiz of quizData">
+  <mat-card *ngIf="quiz?.quizId === quizName">
+    <mat-card-header>
+      <div mat-card-avatar class="header-image"></div>
+      <mat-card-title i18n>{{ quiz?.milestone }} Quiz</mat-card-title>
+      <mat-card-subtitle i18n>
+        <span>How well versed are you in {{ quiz?.milestone }}?</span>
+        <span>Take the quiz and find out!</span>
+      </mat-card-subtitle>
+    </mat-card-header>
+
+    <div mat-card-image class="quiz-topic-image"
+         [ngStyle]="{ 'background': 'url(' + imagePath + quiz?.image + ') no-repeat center top',
```

Comment:
can't this be an img element?

Reply:
changed to img, but getting a light gray border around the image

Reply:
Ok, I'm using img [src]="..." now and it works fine.